src/main/java/duke/taskmanagement/Deadline.java

```diff
@@ -0,0 +1,37 @@
+package duke.taskmanagement;
+import java.time.LocalDate;
+public class Deadline extends Task {
```

Comment:
Perhaps adding some space between these two lines would increase readability and neatness?